R/rev_cohort.R

```diff
@@ -7,16 +7,20 @@
 #' \code{revolver:::CCF.parser} (the default of this parameter).
 #' @param options A list of 2 parameters that should be a boolean value for
 #' \code{ONLY.DRIVER} (use only driver SNVs), and \code{MIN.CLUSTER.SIZE}, the minimum cluster size.
-#' @param annotation A string for annotation of this cohort. This will be prompted
-#' in every print for this object.
+#' @param annotation String for annotation of this cohort. This will be prompted
+#'                   in every print for this object.
 #'
 #' @return An object of class \code{"rev_cohort"}
-#' @export
-#' @import crayon
+#'
+#' @aliases rev_cohort revolver_cohort
```

Comment:
Why you're using an alias for this function? The function `revolver_cohort` is the constructor of the object which usually is called `rev_cohort`

Reply:
The policy was that S3 objects are called `rev_xxx` and exported functions are `revolver_xxx`.